packages/secret-rotator/cloudbuild.yaml

```diff
@@ -0,0 +1,65 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+  # - name: gcr.io/cloud-builders/npm
+  #   id: "build"
+  #   waitFor: ["-"]
+  #   entrypoint: bash
+  #   args:
+  #     - "-e"
+  #     - "./scripts/build-function.sh"
+  #     - "$_DIRECTORY"
+      
+  - name: gcr.io/cloud-builders/docker
+    id: "build-docker"
+    waitFor: ["-"]
+    dir: packages/secret-rotator
+    args:
+      - "build"
+      - "-f"
+      - "Dockerfile"
+      - "-t"
+      - "gcr.io/$PROJECT_ID/secret-rotator"
+      - "."
+
+  - name: gcr.io/cloud-builders/docker
+    id: "push-docker"
+    waitFor: ["build-docker"]
+    args: ["push", "gcr.io/$PROJECT_ID/secret-rotator"]
+
+  - name: gcr.io/cloud-builders/gcloud
+    id: "deploy-cloud-run"
+    waitFor: ["push-docker"]
+    entrypoint: bash
+    args:
+      - "-e"
+      - "./scripts/publish-cloud-run-secret-rotator.sh"
```

Comment:
non blocking suggestion: since this script is not shared, remove unnecessary options (like bucket, key rings) and skip parameterizing the script (move the other values into the script).

Reply:
I missed that from updating the publish cloud run script, thanks.